utils/args/GenericfMRIVolumeProcessingPipeline.py

```diff
@@ -10,6 +10,7 @@
 from collections import OrderedDict
 
 from tr import tr
+from utils.gear_preliminaries import create_sanitized_filepath
```

Comment:
I don't see where this module is.

Reply:
Thanks, @naterichman ,

The reference to the `create_sanitized_filepath` is listed in the "NOTE:" above, and again [here](https://github.com/flywheel-apps/hcp-base/blob/master/utils/gear_preliminaries.py#L320).

Reply:
Totally missed that note!